dev/com.ibm.ws.jca_fat_example_anno/test-resourceadapter/src/com/ibm/example/jca/anno/IndexedRecordImpl.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2022 IBM Corporation and others.
```

Comment:
This is a new class, the copyright should only be 2022

Reply:
Good catch